lib/logger/lib/logger.ex

```diff
@@ -445,6 +445,13 @@ defmodule Logger do
   @levels [:error, :info, :debug] ++ @new_erlang_levels
   @metadata :logger_level
 
+  @doc ~S"""
+  Returns all the available levels.
+  """
+  @doc since: "1.16.0"
+  @spec levels() :: [level()]
```

Comment:
Nit: this will never be empty, so you can do:

```suggestion
  @spec levels() :: [level(), ...]
```

Reply:
Thanks. Updated.